package.json

```diff
@@ -46,6 +47,7 @@
     "swagger-ui-express": "^5.0.0",
     "tsoa": "^6.2.1",
     "tsyringe": "^4.8.0",
+    "undici": "^6.16.1",
```

Comment:
undici is already a dev dependency, you should be able to just remove this line and `npm i`